lm_eval/tasks/nq_open/README.md

```diff
@@ -0,0 +1,22 @@
+### Paper
+
+Question Answering dataset based on aggregated user queries from Google Search.
+
+Homepage: https://research.google/pubs/natural-questions-a-benchmark-for-question-answering-research/
+
+Paper: https://storage.googleapis.com/gweb-research2023-media/pubtools/pdf/1f7b46b5378d757553d3e92ead36bda2e4254244.pdf
```

Comment:
This variant was actually introduced by https://github.com/google-research-datasets/natural-questions/tree/master/nq_open --sorry for the confusion.

https://aclanthology.org/P19-1612/ is the accompanying paper